completion/graphicsonthefly.cwl

```diff
@@ -0,0 +1,107 @@
+# graphicsonthefly package
+# ponte-vecchio 2023-01-17 for v0.9.6
+
+#include: graphicx
+
+#keyvals:\usepackage/graphicsonthefly#c
+renewall
+animate
+all
+curl
+nocoalesce
+convert
+#endkeyvals
+
+## Animations
+#ifOption:all
+#include: animate
+\animatedgifonthefly[%<options%>%keyvals]{%<fps%>}{%<image name%>}{%<start%>}{%<end%>}{%<url%>}
```

Comment:
The `%keyvals` in `%<options%>%keyvals` is superfluous since `%<options%>` alone is specially recognized for keyvals

Reply:
Reflected in the commit, changed all `%<options%>%keyvals` to just `%<options%>`